index.d.ts

```diff
@@ -68,18 +68,15 @@ declare namespace CliTable3 {
         readonly width: number;
     }
 
-    type Table = HorizontalTable | VerticalTable | CrossTable;
+    type Table = GenericTable<HorizontalTableRow|VerticalTableRow|CrossTableRow>;
     type Cell = CellValue | CellOptions;
 
-    type HorizontalTable = GenericTable<HorizontalTableRow>;
     type HorizontalTableRow = Cell[];
 
-    type VerticalTable = GenericTable<VerticalTableRow>;
     interface VerticalTableRow {
         [name: string]: Cell;
     }
 
-    type CrossTable = GenericTable<CrossTableRow>;
```

Comment:
removing these types should be considered a breaking change IMHO. can you please add them back?

Reply:
@DanielRuf Thoughts? I am in no position to comment.

Reply:
> removing these types should be considered a breaking change IMHO. can you please add them back?

I agree here. Can we keep this type somehow?